h_max: 103
h_min: 91
s_max: 255
s_min: 65
v_max: 255
v_min: 63
